hnn_core/cell.py

```diff
@@ -239,6 +242,13 @@ def record_current_soma(self):
                 " but no self.synapses dict was found")
             pass
 
+    def record_voltage_soma(self):
+        """Record current at soma."""
+        if self.record_vsoma:
+            self.rec_v = h.Vector().record(self.soma(0.5)._ref_v)
+        else:
+            self.rec_v = h.Vector()
```

Comment:
we're not super consistent but I believe we should document all the attributes too.

Also, it would be preferable to have `self.rec_v = h.Vector()` in the init so that here you can simply do:

```py
self.rec_v.record(....)
```